src/Sylius/Behat/Service/NotificationChecker.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Service;
+
+use Sylius\Behat\Exception\NotificationExpectationMismatchException;
+use Sylius\Behat\Service\Accessor\NotificationAccessorInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+final class NotificationChecker implements NotificationCheckerInterface
+{
+    /**
+     * @var NotificationAccessorInterface
+     */
+    private $notificationAccessor;
+
+    /**
+     * @param NotificationAccessorInterface $notificationAccessor
+     */
+    public function __construct(NotificationAccessorInterface $notificationAccessor)
+    {
+        $this->notificationAccessor = $notificationAccessor;
+    }
+
+    /**
+     * @param string $resource
+     *
+     * @throws NotificationExpectationMismatchException
+     */
+    public function checkCreationNotification($resource)
+    {
+        $message = sprintf('Success %s has been successfully created.', $this->humanizeResourceName($resource));
```

Comment:
Unfortunately, everything is correct. 
![zrzut ekranu 2016-03-31 o 08 58 43](https://cloud.githubusercontent.com/assets/6213903/14167946/dc1164ba-f71e-11e5-873d-31f759c69ce7.png)